tests/sla_tests.js

```diff
@@ -82,10 +84,42 @@ var getMockClient = function() {
         cb(null, bridges[bridges.length-1]);
       }
     };
-    this.Channel= function() {
+    this.Channel = function() {
       var newChan = getMockChannel();
       return newChan;
     };
+    this.deviceStates = {
+      update: function(param, cb) {
```

Comment:
'param' should really be 'params' as this is an object. Another common name for such objects in JavaScript is 'options'.